StratisCore.UI/src/app/app.component.ts

```diff
@@ -56,8 +58,18 @@ export class AppComponent implements OnInit, OnDestroy {
 
     this.subscription = stream$.subscribe(
       (data: NodeStatus) => {
-        this.loading = false;
-        this.router.navigate(['login']);
+        this.apiConnected = true;
+        this.statusIntervalSubscription = this.apiService.getNodeStatusInterval(true)
+          .subscribe(
+            response =>  {
+              const statusResponse = response.featuresData.filter(x => x.namespace === 'Stratis.Bitcoin.Base.BaseFeature');
+              if (statusResponse[0].state === 'Initialized') {
+                this.loading = false;
+                this.statusIntervalSubscription.unsubscribe();
```

Comment:
Do we need this? Can't we just do `this.apiService.getNodeStatusInterval(true).pipe(take(1))` instead?

Reply:
Not sure, I'll test.

Reply:
Doesn't do the trick, will leave it as is and add unsubscribe at onDestroy.
Thanks!